Regula_Falsi_Method/regula_falsi.cpp

```diff
@@ -0,0 +1,71 @@
+/* Girlscript Summer Of Code
+Regula Falsi Method in c
+In mathematics, the false position method or regula falsi is a very old method for solving an equation in one unknown,
+that, in modified form, is still in use.Regula Falsi method or the method of false position is a numerical method for 
+solving an equation in one unknown. It is quite similar to bisection method algorithm and is one of the oldest approaches.
+It was developed because the bisection method converges at a fairly slow speed. In simple terms, the method is the
+trial and error technique of using test ("false") values for the variable and then adjusting the test value according
+to the outcome.
+In simple terms, the method is the trial and error technique of using test ("false") values for the variable and
+then adjusting the test value according to the outcome. This is sometimes also referred to as "guess and check".
+Versions of the method predate the advent of algebra and the use of equations.*/
```

Comment:
one line should contain maximum 80 characters, break this line according to that
I feel  it's better if you make the commend short